dev/breeze/src/airflow_breeze/params/doc_build_params.py

```diff
@@ -42,6 +55,9 @@ def args_doc_builder(self) -> list[str]:
             doc_args.append("--one-pass-only")
         if AIRFLOW_BRANCH != "main":
             doc_args.append("--disable-provider-checks")
+        # if short_version is set, the self.package_filter is preprocessed
```

Comment:
I think it would be better to use the `argument` instead of `--short-version`. And you already have this one:

```
argument_packages_plus_all_providers = click.argument(
    "packages_plus_all_providers",
    nargs=-1,
    required=False,
    type=BetterChoice(["all-providers"] + get_available_documentation_packages(short_version=True)),
)
```

See where it us used.

The advantage of using argument over option is tha tyou can use it without anything like this:

```
breeze build.docs cncf.kubernetes apache-airflow sftp
```

Reply:
Sounds good. Making some modifications to accomodate the situation. I think we should not retire the older behaviour too. Hence, we support the short hands as arguments but `--package-filter` remains. Trying this out